plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeSplit.java

```diff
@@ -65,6 +66,33 @@ public DeltaLakeSplit(
             @JsonProperty("splitWeight") SplitWeight splitWeight,
             @JsonProperty("statisticsPredicate") TupleDomain<DeltaLakeColumnHandle> statisticsPredicate,
             @JsonProperty("partitionKeys") Map<String, Optional<String>> partitionKeys)
+    {
+        this(
+                path,
+                start,
+                length,
+                fileSize,
+                fileRowCount,
+                fileModifiedTime,
+                deletionVector,
+                ImmutableList.of(),
+                splitWeight,
+                statisticsPredicate,
+                partitionKeys);
+    }
+
+    public DeltaLakeSplit(
```

Comment:
We shouldn't be adding a new constructor, just update the existing calls to constructor